aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -426,17 +433,17 @@ module aptos_framework::genesis {
             rewards_rate_denominator,
             voting_power_increase_limit
         );
-
+        features::change_feature_flags(aptos_framework, vector[1, 2], vector[]);
```

Comment:
What is this for?

Reply:
It's for `initialize_features`: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/vm-genesis/src/lib.rs#L393